ui/packages/keyvis/rollup.config.js

```diff
@@ -1,72 +1,18 @@
-import typescript from 'rollup-plugin-typescript2'
-import createTransformer from 'ts-import-plugin'
-import commonjs from 'rollup-plugin-commonjs'
-import external from 'rollup-plugin-peer-deps-external'
-import postcss from 'rollup-plugin-postcss'
-import resolve from 'rollup-plugin-node-resolve'
-import url from 'rollup-plugin-url'
-import svgr from '@svgr/rollup'
-import yaml from '@rollup/plugin-yaml'
-
 import pkg from './package.json'
-
-// https://github.com/Brooooooklyn/ts-import-plugin
-const transformer = createTransformer({
-  libraryDirectory: 'es',
-  libraryName: 'antd',
-  style: true,
-})
+import baseConfig from '../rollup.config.base'
 
 export default {
-  input: 'src/index.ts',
   output: [
     {
       file: pkg.main,
       format: 'cjs',
-      exports: 'named',
       sourcemap: true,
     },
     {
       file: pkg.module,
       format: 'es',
-      exports: 'named',
       sourcemap: true,
     },
   ],
```

Comment:
how about:

```js
export default function baseConfig(pkg) {
    return {
        ...
    }
}
```

when using:

```js
import pkg from './package.json'
import baseConfig from '../rollup.config.base'
export default baseConfig(pkg)
```

Reply:
Good idea! will update it soon after adding the tooltip for the column header.